plugins/catalog-backend/src/service/createRouter.ts

```diff
@@ -113,18 +116,56 @@ export async function createRouter(
   }
 
   if (refreshService) {
+    // TODO: Potentially find a way to track the ancestor that gets refreshed to refresh this entity (as well as the child of that ancestor?)
     router.post('/refresh', async (req, res) => {
       const { authorizationToken, ...restBody } = req.body;
 
-      const credentials = authorizationToken
-        ? await auth.authenticate(authorizationToken)
-        : await httpAuth.credentials(req);
+      try {
+        await auditor.info({
+          eventName: 'CatalogEntityRefresh',
+          message: 'Refreshing entity with ref',
+          status: 'unknown',
+          stage: 'initiation',
+          meta: {
+            entityRef: restBody.entityRef,
+          },
+          request: req,
+        });
```

Comment:
Thinking a bit more about this I think we can likely have an API that allows for reporting the outcome, and then it's up to the implementation to decide whether to record it or not. Same thing could go for read actions etc. although we'd need some way to signal those at that point, which could be what we replace levels with. Thinking for example `auditor.logRead`, `auditor.logWrite`, and something like `auditor.logAdminAction`, although that can probably come later since we don't really have that yet.

Reply:
How do you feel about the API used for the `/refresh` endpoint? I believe this update significantly cuts down on boilerplate while still providing the flexibility to call `.fail()` multiple times when needed.

Reply:
Yep, love it! 👍